api/src/main/java/org/codice/ddf/admin/api/handler/commons/HandlerCommons.java

```diff
@@ -22,15 +22,25 @@
 
```

Comment:
I'm indifferent on this. We may want to add functionality in the future, when we'd have to rename again.
Commons is not inaccurate, just imprecise.